src/main/java/org/folio/des/controller/JobsController.java

```diff
@@ -53,6 +53,16 @@ public ResponseEntity<Job> upsertJob(@RequestHeader("X-Okapi-Tenant") String ten
     return new ResponseEntity<>(service.upsertAndSendToKafka(job, true), job.getId() == null ? HttpStatus.CREATED : HttpStatus.OK);
   }
 
+  @Override
+  public ResponseEntity<Void> resendExportedFile(UUID id) {
+  Job job = service.get(id);
+    if (isMissingRequiredParameters(job)) {
```

Comment:
We can omit this validation for resend endpoint, it just EDIFACT functionality, and isMissingRequiredParameters checks BULK_EDIT/E_HOLDINGS functionality, so lets delete this if

Reply:
fixed